### DISCUSSION TITLE
hey can you put all pluto tv channels in one link?

### DISCUSSION THREAD

**Discussion by supertails12** [iptv-org/community#325] [Q&A] [answered]
hey can you put all pluto tv channel in one m3u link? would make it easier to watch thank you!

**Comment by supertails12** [comment]
also add the latest pluto tv channels

**Comment by Sphinxroot** [comment, marked as answer]
Currently  they are splited by country  and are easy to maintain ,  but if you wanna use the all in one , use  https://i.mjh.nz/PlutoTV/all.m3u8 in your player
or
you may also use Matt Huisman ,  Slyguy repo /  Pluto TV addon in kodi  , mini epg is also provided in to it
